app/assets/javascripts/playlist.js

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
since we've moved all things to webpacker, this needs to go in `./app/javascript/src/js`, and be imported in `./app/javascript/packs/application.js`.